design/sketch.webidl

```diff
@@ -640,7 +640,7 @@ dictionary WebGPUSwapChainDescriptor {
 
 interface WebGPUSwapChain {
     void configure(WebGPUSwapChainDescriptor descriptor);
-    WebGPUTexture getNextTexture();
+    WebGPUTexture acquireNextTexture();
```

Comment:
Within one RAF callback, `getNextTexture` should always return the same texture.  To that end, I think "get" instead of "acquire" seems like the correct terminology.

Separately, after calling `present`, I propose that `getNextTexture` should return null to discourage web developers to putting the foreground thread into an infinite loop.

Reply:
If it returns the same texture multiple times, I agree. But it could also be an error/null to call it in such a way that you get the same texture again.

I think how this should work is that immediately after the animation frame task (or whatever it's called - the task that composites the page), there should be exactly one texture available. After that, you can either get (idempotent) or acquire (one-time) the texture. Acquire would leave exactly zero textures available (returning null or exception).

Reply:
Why not "acquireCurrentTexture" which is idempotent if called multiple times in the same rAF()? (Or idempotent if called multiple times between adjacent present() calls?)

Reply:
"Acquire" implies to me that I (the JS app) now have the thing, and the API (browser) does not have it anymore.

If the texture is indeed tied to animation frames (as I described above and in 182), I am partial to "current" over "next".